src/wallet/rpcwallet.cpp

```diff
@@ -135,14 +135,15 @@ UniValue getnewaddress(const JSONRPCRequest& request)
         return NullUniValue;
     }
 
-    if (request.fHelp || request.params.size() > 1)
+    if (request.fHelp || request.params.size() > 2)
         throw std::runtime_error(
-            "getnewaddress ( \"account\" )\n"
+            "getnewaddress ( \"account\" ) ( \"style\" )\n"
```

Comment:
I think you want `"getnewaddress ( \"account\" \"style\" )\n"`

Reply:
Fixed.